packages/teleterm/src/services/tshdEvents/index.ts

```diff
@@ -0,0 +1,79 @@
+import { EventEmitter } from 'node:events';
```

Comment:
This did start as a separate file but then I was like… they're small enough that they can live together in the same file for now. And I wanted to keep the tidy structure of `packages/teleterm/src/services` where it has directories only _and_ I wanted to call just `import sth from 'teleterm/services/tshdEvents'` in other files.

And since there's no other files in that directory, I just kept them in `index.ts`.

If I renamed that to `tshdEvents.ts` then I would need to move it up a lavel to `packages/teleterm/src/services/tshdEvents.ts` to keep the same import path, wouldn't I?

After all, it's not that big of an issue, I can just do that.

Reply:
Ok, let it stay as it is.
In the future `createTshdEventsService()` will grow probably, so then it will make sense to split it into more files.